src/query/graphite/native/builtin_functions.go

```diff
@@ -2448,8 +2453,9 @@ func init() {
 		3: "",    // fname
 	})
 	MustRegisterFunction(sumSeries)
-	MustRegisterFunction(sumSeriesWithWildcards)
-	MustRegisterFunction(aggregateWithWildcards)
```

Comment:
Hm don't think you meant to remove this function being registered no? `aggregateWithWildcards` is being removed with this change.